Mathlib/Analysis/Complex/UpperHalfPlane/Basic.lean

```diff
@@ -139,6 +139,40 @@ theorem ne_zero (z : ℍ) : (z : ℂ) ≠ 0 :=
   mt (congr_arg Complex.im) z.im_ne_zero
 #align upper_half_plane.ne_zero UpperHalfPlane.ne_zero
 
+end UpperHalfPlane
+
+namespace Mathlib.Meta.Positivity
+
+open Lean Meta Qq
+
+/-- Extension for the `positivity` tactic: `UpperHalfPlane.im`. -/
+@[positivity UpperHalfPlane.im _]
+def evalUpperHalfPlaneIm : PositivityExt where eval {u α} _zα _pα e := do
+  -- TODO: can't merge the `match`es without lean4#3060
```

Comment:
Do I understand correctly that https://github.com/leanprover/lean4/pull/3060 is now in Lean master but we still need to wait for Mathlib to pick up this version of Lean so this PR is currently blocked?

Reply:
No, this PR isn't blocked by that; this TODO comment doesn't need to be addressed immediately, it's just a record of future cleanup.